sample-apps/ruby-rails-sample-app/config/initializers/opentelemetry.rb

```diff
@@ -0,0 +1,23 @@
+require 'opentelemetry/sdk'
+require 'opentelemetry/exporter/otlp'
+require 'opentelemetry/instrumentation/all'
+
+OpenTelemetry::SDK.configure do |c|
+    c.service_name = 'ruby-sample-app'
+  
+    c.id_generator = OpenTelemetry::Propagator::XRay::IDGenerator
+    c.propagators = [OpenTelemetry::Propagator::XRay::TextMapPropagator.new]
+  
+    c.add_span_processor(
+      OpenTelemetry::SDK::Trace::Export::BatchSpanProcessor.new(
+        OpenTelemetry::Exporter::OTLP::Exporter.new(
+          endpoint: 'http://0.0.0.0:4318/v1/traces'
+        )
+      )
+    )
+  
+    # Alternatively, we could just enable all instrumentation:
+    c.use_all()
```

Comment:
If you remove the dependencies to `-all`, initialize only the libraries that you are going to use.

https://opentelemetry.io/docs/instrumentation/ruby/automatic/#configuring-specific-instrumentation-libraries

Reply:
Done